c2corg_ui/templates/helpers/view.html

```diff
@@ -390,7 +390,7 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
 
 <%def name="get_document_min_max(document, prop)">\
   % if document.get(prop + '_min') and document.get(prop + '_max') and (document.get(prop + '_max') is not None and document.get(prop + '_min') is not None):
-  <p><b x-translate class="value-title">${prop} min / max </b>: <span class="value"> ${document[prop + '_min']}m / ${document[prop + '_max']}m</span></p>
+  <p><b x-translate class="value-title">${prop} min/max </b>: <span class="value"> ${document[prop + '_min']}m / ${document[prop + '_max']}m</span></p>
```

Comment:
Shouldn't you translate ``${prop}`` and ``min/max`` separately?
I guess it would help reducing the number of strings to be translated.